fall18/p2/test.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+import subprocess, json, re, sys
+from inspect import getmembers, isfunction
+import main
+
+PASS = 'PASS'
+EPSILON = 0.0001
+
+TESTS_FUNC = [
+    ("getMaximumLand", ["Arizona", "California", "Connecticut"], 163694.74), #1
+    ("getMinimumPopulationDensity", ["Wisconsin", "Iowa", "Minnesota", 2010], 54.13546968775862), #2
+    ("predictPopulation", ["Hawaii", 2000, 2010, 0.4], 66147678.89670547), #3
+    ("predictPopulation", ["Hawaii", 2000, 2010], 3293299.011605786), #4
+    ("calcGrowthRate", ["Hawaii", 2000, 2015], 0.010596535979501064) #5
+]
+
+TESTS_PRINT = [
+    ["The area of Wisconsin is ", 65496.38], #1
+    ["The population of Wisconsin in 2010 is ", 5686986.0], #2
+    ["Maximum land area among Wisconsin, Iowa, Minnesota is ", 86935.83], #3
+    ["Minimum population density among Wisconsin, Iowa, Minnesota is ", 52.002450206414075], #4
+    ["The predicted population for Wisconsin in year 2010 is (assume start yearA is 2000, growth rate is 0.5) ", 796039951.1495125], #5
+    ["The growth rate for Wisconsin between year 2000 and 2010 is ", 0.005853103209551789] #6
+]
+
+TESTS_FUNC_NUM = len(TESTS_FUNC)
+TESTS_PRINT_NUM = len(TESTS_PRINT)
+PROBLEMS = TESTS_FUNC_NUM + TESTS_PRINT_NUM
```

Comment:
It's confusing to me that this is 11 but there are 16 tests.  I'm struggling to map the failures to specific tests...

Reply:
I only have 11 tests here. What's  16 comes from?